### ISSUE TITLE
Some button-ish functionality is implemented as <span onClick={}>

### ISSUE THREAD

**Issue by brian-c** [zooniverse/Panoptes-Front-End#1469]
These should be `<button>`s for accessibility and dev legibility, but there are some style specificity issues (`.talk button`) to work out first.

Noticed in #1436.

**Comment by eatyourgreens** [comment]
Related to this, when a `<button>` element's content is an icon, can we also add an `aria-label` attribute with the text label for the button?

**Comment by chrissnyder** [comment]
Looked to be all converted. Reopen if I missed some.
